src/lib/snack-bar/snack-bar.ts

```diff
@@ -64,6 +62,10 @@ export class MdSnackBar {
     } else {
       snackBarRef.containerInstance.enter();
     }
+    if (config.autoHide > 0) {
+      setTimeout(() => snackBarRef.dismiss(), config.autoHide + 225);
```

Comment:
Why is the `225` baked in here? (I'm assuming it's the animation duration, but doesn't dismiss handle that already?)